Makefile

```diff
@@ -60,10 +60,17 @@ endif
 .PHONY: lint
 lint: lint-go lint-proto lint-charts lint-ui
 
+.PHONY: format
+format: format-go format-ui
+
```

Comment:
This is a nice addition. I am curious if something specific prompted this. I take it for granted that our editors format things appropriately, but with an uptick in community contributions, we're probably seeing a wider variety of editors, editor plugins, etc. It's nice to have an editor-agnostic final arbiter of correct formatting.

I wonder if we should consider putting a formatting check into the CI process. We do something similar already for codegen. We re-run codegen and check that it produces no discrepancies with the generated code in the PR. We could tackle that in a follow-up, of course.

Reply:
It was actually something I've been meaning to add for a long time, and ended up scratching the itch while working on this PR 😄 

For the UI, we already have a formatting check in CI ✅ . I thought there was one for the Go code too, but maybe I'm misremembering

Reply:
@rbreeze we lint Go code in CI, but I'm not positive the linter catches all formatting issues.

In general, lint == stop you from making common mistakes, but that sometimes does overlap with formatting issues.